app/controllers/application_controller.rb

```diff
@@ -2,13 +2,14 @@
 
 class ApplicationController < ActionController::Base
   before_action :store_request_id
+  before_action :assign_sentry_contexts
 
   def store_request_id
     RequestStore.store[:request_id] = request.uuid
   end
 
   def assign_sentry_contexts
-    Raven.extra_context(request_id: RequestStore.store[:request_id])
+    Raven.tags_context(request_id: RequestStore.store[:request_id])
```

Comment:
I don't see a test for this. Is it testable?

Reply:
it has coverage